questionnaire/questionnaire/utils/taskRunner/tasks/transformAndUpload/index.js

```diff
@@ -132,14 +132,21 @@ function transformQuestionnaire(questionnaire) {
         sectionSchema.properties['p-check-your-answers'].properties.summaryInfo.summaryStructure;
     flattenAnswers(themeContent);
 
-    return {
+    const answers = questionnaire.getAnswers();
+    const transformedQuestionnaire = {
         meta: {
-            caseReference: questionnaire.getAnswers().system['case-reference'],
-            funeralReference: questionnaire.getAnswers().system['secondary-reference']
+            caseReference: answers.system['case-reference'],
+            funeralReference: answers.system['secondary-reference']
         },
         themes: themeContent,
         declaration: getDeclaration(questionnaire)
     };
+
+    if (answers.origin) {
+        transformedQuestionnaire.meta.channel = answers.origin.channel;
```

Comment:
Should we keep a reference to `origin` here? 🤔

Reply:
The transformed questionnaire is a flattened view, I was naming this on the basis that the bit of data we're interested in is the "channel" value, in the same way we lose reference to "system" when mapping "caseReference" above.

Reply:
makes absolute sense! 👍